### PR TITLE
add tesla to supplypacks

### PR DESCRIPTION
**Pull request by Robertha89**
:cl: ppi
rcsadd: Adds tesla engine to supplypacks
rcsadd: Adds grounding rod to supplypacks
rcsadd: Adds tesla coil to supplypacks
/:cl:

### CODE REVIEW

**Comment by Crazylemon64** [comment]
:+1:

**Comment by Crazylemon64** [comment]
You may want a grounding rod supply pack, too, keep in mind.

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -524,6 +524,12 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
 	cost = 10
 	containername = "singularity generator crate"
 
+/datum/supply_packs/engineering/engine/tesla
+	name = "Energy Ball Generator"
```
Comment (Markolie) [inline code/datums/supplypacks.dm]: Add Crate to the end please.

File: code/datums/supplypacks.dm [hunk 2]
```diff
@@ -524,6 +524,12 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
...
+	contains = list(/obj/machinery/the_singularitygen/tesla)
+	cost = 10
+	containername = "energy ball generator"
```
Comment (Markolie) [inline code/datums/supplypacks.dm]: Same here.

**Comment by Gummin** [comment]
:+1:

**Pull request merged**